.gitignore

```diff
@@ -1,2 +1,5 @@
 Cargo.lock
-/target/
+
+# Build directories
+# Without a trailing slash to include symbolic links.
```

Comment:
Let's keep it simple. The rationale can be in the commit description.

```suggestion
```

Reply:
This is not obvious. You confirmed it with your response ;)

From my experience rationale and decisions should be commented directly in the code. No one ever reads commit messages.

Reply:
It should not be obvious that using a target symlink is possible. That's quite a corner-case. And I don't think it should be advertised. That's something done because it's needed, not because it's possible. If not possible, a PR is created like here.

Regarding documentation, it's a trade-off between signal and noise. In this particular case, it's adding more noise than signal in my opinion. The only signal would be to prevent someone to revert to directory only (this has very low probability to happen). The noise is more than 33% of the file (60% in the current state of the PR) for something you rarely need to know.

Reply:
Disagree and commit ;)

Reply:
Yes, I think it's fine to disagree on such trivial matter. Thanks again for the PR!